src/Core/Form/IdentifiableObject/DataHandler/SqlRequestFormDataHandler.php

```diff
@@ -41,19 +44,36 @@ final class SqlRequestFormDataHandler implements FormDataHandlerInterface
      */
     private $commandBus;
 
+    /**
+     * @var SqlRequestFormDataValidator
+     */
+    private $requestSqlFormDataValidator;
+
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
     /**
      * @param CommandBusInterface $bus
      */
-    public function __construct(CommandBusInterface $bus)
-    {
+    public function __construct(
```

Comment:
Could you add behat tests for this behavior change ?

Reply:
I add simple behats tests for the form handling SELECT, UPDATE, DELETE, INSERT, Bad query, and unknown table.

Reply:
:+1: